projects/js-packages/publicize-components/src/components/social-previews/modal.scss

```diff
@@ -15,7 +15,6 @@ $highlight-color: #3858e9;
 
 	.components-modal__content {
 		padding: 0;
-		overflow: hidden;
```

Comment:
Allow vertical scrolling of the modal content.

Reply:
[This was introduced](https://github.com/Automattic/jetpack/pull/30812/files#diff-a001514b4d0a54981416637f6376154ad2d76e3763c597d1af136084502598d4R18) as part of #30812 which fixed a problem where you can scroll the header. It's fine to remove, but we'd need to find another fix for that issue.